test/services/MeasureService.test.ts

```diff
@@ -345,6 +346,74 @@ describe('MeasureService', () => {
         });
     });
   });
+
+  describe('$data-requirements', () => {
+    // spy on calculation function
+    jest.spyOn(Calculator, 'calculateDataRequirements').mockResolvedValue({
+      results: {
+        resourceType: 'Library',
+        type: {
+          coding: [{ code: 'module-definition', system: 'http://terminology.hl7.org/CodeSystem/library-type' }]
+        },
+        status: 'draft',
+        dataRequirement: []
+      }
+    });
+
+    it('returns 200 and a Library for a simple measure with dependencies and no period params', async () => {
+      await supertest(server.app)
+        .post('/4_0_1/Measure/$data-requirements')
+        .send({ resourceType: 'Parameters', parameter: [{ name: 'id', valueString: 'testWithRootLibAndDeps' }] })
+        .set('content-type', 'application/fhir+json')
+        .expect(200)
+        .then(response => {
+          expect(response.body.resourceType).toEqual('Library');
+          expect(response.body.dataRequirement).toHaveLength(0);
+        });
+    });
+
+    it('returns 200 and a Library for a request with id in the url', async () => {
+      await supertest(server.app)
+        .post('/4_0_1/Measure/testWithRootLibAndDeps/$data-requirements')
+        .set('content-type', 'application/fhir+json')
```

Comment:
```suggestion
        .get('/4_0_1/Measure/testWithRootLibAndDeps/$data-requirements')
```
I think we can do this since nothing is being passed through the body (similar to test on line 163 for Measure/$package).